.github/workflows/CI.yml

```diff
@@ -19,11 +19,12 @@ jobs:
       matrix:
         # we test on all versions back to LTS because we want to test for stability across
         # julia versions
-        version: 
+        version:
           - '1.6' # LTS
           - '1.7'
           - '1.8'
           - '1.9'
+          - '1.10'
```

Comment:
Test also on nightly to be alerted of breakages _before_ the release of julia and without having to wait for someone to add the `1.11` line?